pkg/util/ipset/ipset.go

```diff
@@ -48,6 +50,7 @@ type Interface interface {
 	ListSets() ([]string, error)
 	// GetVersion returns the "X.Y" version string for ipset.
 	GetVersion() (string, error)
+	// TODO: add comment message for ipset
```

Comment:
I dont understand this TODO. What are you referring to?

Reply:
All set types support the optional comment extension. Enabling this extension on an ipset enables you to annotate an ipset entry with an arbitrary string. This string is completely ignored by both the kernel and ipset itself and is purely for providing a convenient means to document the reason for an entry's existence. 

For example, the following shell command is illegal:
ipset add foo 1.1.1.1 comment `this comment is for something`

I just came up an idea about that but I am not sure if we really need it.

I can remove this line for now and will fire another PR if needed.

Reply:
I just pushed an update to remove this TODO.